Makefile

```diff
@@ -228,16 +229,14 @@ gitserver:
 # ref: https://stackoverflow.com/questions/33509194/command-to-delete-all-pods-in-all-kubernetes-namespaces
 .PHONY: e2e-test-clean
 e2e-test-clean:
-	$(KUBECTL) delete -n numaplane-e2e isbsvc --all
```

Comment:
removed as e2e test run on CI was hanging on deleting isbsvc / pipeline. unsure why as of now.

Reply:
Are you deploying a Controller anywhere? If so, I know that would've been the original motivation for including these lines. 

If you do take this out, then the comment on lines 225-226 needs to be removed as well

Reply:
We are still deploying the controller in `TestNumaflowGitSync`. I was seeing an issue where the delete would hang on deleting the ISB or Pipeline which is odd cause locally those resources don't usually exist after all the tests have run. I will check tomorrow to see but otherwise I could remove the comment as well.

Reply:
If you're still deploying a Controller, then maybe you do need those lines? If so, if so is it possible to maybe add logging in the CI to figure out what's happening? The only thing I can think of is that the Controller somehow was previously in a state to add the Finalizer, but is no longer in a state to remove the Finalizer

Reply:
I can try something like that, from what I can see on the CI the controller is already long deleted but the pl/isbsvc still exist which is not the case when running locally. I could probably add a line to remove the finalizers if thats the case.

Reply:
yeah, it seems okay if you want to remove the finalizers through patch or something

Reply:
That is the most likely what i'll do for now. Can get that added today.

Reply:
is it possible for the test case to clean up the state with order to avoid the CI to do so. that may be easier to control?

Reply:
Yes, in the test case there could a call to remove all files from the repo thus deleting the resources.

Reply:
Giving this approach a try now.